lib/rmt/scc.rb

```diff
@@ -23,6 +23,11 @@ def sync
     data.each { |item| create_product(item) }
     data.each { |item| migration_paths(item) }
 
+    # NOTE: bug introduced in SCC where some non-SUSE products with negative IDs
+    # were introduced. This should never have happened since they are managed
+    # elsewhere. Purge them now if this is the case for this registration proxy.
+    Product.where('id < ?', 0).destroy_all
```

Comment:
How about adding this as a migration, so we don't have keep it in our codebase?

Reply:
Ah, of course!